### DISCUSSION TITLE
Developing a different UI style

### DISCUSSION THREAD

**Discussion by LunaticMuch** [Pagefind/pagefind#540] [Q&A]
I am using the pagefind-ui on my astro website (which also include tailwind). The look and feel with the pagefind UI is not great. I would much prefer to define my elements, but apart from rewriting all the styles in `ui.css` what other option I could potentially look at?

**Comment by mikecee** [comment]
I had a similar question. There have been a few references to an [upcoming modular UI](https://www.npmjs.com/package/@pagefind/modular-ui) which sounds like it will address this type of problem. Now to find the time to experiment...

**Comment by bglw** [comment]
Hello! Sorry that I missed this.

Yes, currently providing your own CSS is the best method to restyle. Pagefind tries to use good elements under the hood, so without CSS it hopefully slots into existing layouts moderately well.
